app/services/touch-bar.ts

```diff
@@ -28,14 +29,16 @@ export class TouchBarService extends Service {
   micBar: electron.TouchBar;
   micSlider: electron.TouchBarSlider;
   micMute: electron.TouchBarButton;
-  micUnmutedIcon = electron.remote.nativeImage.createFromNamedImage(
-    'NSTouchBarAudioInputTemplate',
-    [0, 0, 1],
-  );
-  micMutedIcon = electron.remote.nativeImage.createFromNamedImage(
-    'NSTouchBarAudioInputMuteTemplate',
-    [0, 0, 1],
-  );
+  micUnmutedIcon = remote.nativeImage.createFromNamedImage('NSTouchBarAudioInputTemplate', [
+    0,
+    0,
+    1,
+  ]);
+  micMutedIcon = remote.nativeImage.createFromNamedImage('NSTouchBarAudioInputMuteTemplate', [
+    0,
+    0,
+    1,
+  ]);
```

Comment:
lmao i hate this auto formatting so much

Reply:
yeah it's not great